server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraBlobModule.java

```diff
@@ -65,5 +67,17 @@
             .addPartitionKey(BucketBlobParts.ID, DataType.text())
             .addClusteringColumn(BucketBlobTable.NUMBER_OF_CHUNK, DataType.cint()))
 
+        .table(BlobTables.DumbBlobCache.TABLE_NAME)
```

Comment:
For this, because module cache are separated from last PR, for testing i added it here. I dont know how to test it in separated way.Pls help me on that.

Reply:
Please write a CassandraDumbBlobStoreCacheModule

Reply:
I mean, the problem is when you had 2 Module classes, how can you do registry extension?

Reply:
You can use CassandraModule.aggregate in your tests

Reply:
Can we remove this table declaration from here?

Reply:
sorry, this think are happened by wrong rebase process.